hw/ip/otbn/doc/_index.md

```diff
@@ -356,9 +356,10 @@ The `L`, `M`, and `Z` flags are determined based on the result of the operation
 
 ### Loop Stack
 
-The LOOP instruction allows for nested loops; the active loops are stored on the loop stack.
-Each loop stack entry is a tuple of loop count, start address, and end address.
-The number of entries in the loop stack is implementation-dependent.
+OTBN has two instructions for hardware-assisted loops: [`LOOP`]({{< relref "hw/ip/otbn/doc/isa#loop" >}}) and [`LOOPI`]({{< relref "hw/ip/otbn/doc/isa#loopi" >}}).
+Both use the same state for tracking control flow.
+This is a stack of tuples containing a loop count, start address and end address.
+The stack has a maximum depth of eight and the top of the stack is the current loop.
```

Comment:
In RTL the stack and the current loop are separate things, but this is an implementation detail not worth discussing here. It does mean the effective loop stack size is 9 though. Happy to switch the internal stack size to 7 so an effective stack of 8.

Reply:
PR here to address this: https://github.com/lowRISC/opentitan/pull/4510

Reply:
Cool. I reckon that's a detail that probably doesn't belong in the instruction docs. Let's discuss the correct loop stack size in #4510.